modules/ROOT/pages/dataweave-cookbook.adoc

```diff
@@ -149,6 +140,31 @@ the `..*` descendants selector on a key to select values from a flattened list.
 | Defines a function that expects to receive two inputs: a function to apply and an element to apply it on. The function is also recursively applied to the element's children. It uses `mapObject` `lower` `is` `if/else`.
 |===
 
+
+== DataWeave Examples in Mule
```

Comment:
Mule Examples that Implement DataWeave? 
Example DataWeave Transformations in Mule Applications? 
Examples of Mule Applications that Implement DataWeave?